File: HealthCatalyst.Models/Interest.cs
using HealthCatalyst.Interfaces.Models;

namespace HealthCatalyst.Models
{
    /// <summary>
    /// Implementation of <see cref="IInterest"/>
    /// </summary>
    public class Interest : IInterest
    {
        public int InterestId { get; set; }
        public int? PersonId { get; set; }
        public string Description { get; set; }
    }
}

File: HealthCatalyst.Web/ClientApp/src/app/models/interest.ts
import { IInterest } from '../interfaces/iinterest';

export class Interest implements IInterest {
    constructor(interestId: number, personId: number, desc: string){
        this.interestId = interestId;
        this.personId = personId;
        this.description = desc;
    }
    interestId: number;
    personId: number;
    description: string;
}
